src/libs/EmojiUtils.ts

```diff
@@ -235,37 +234,6 @@ function mergeEmojisWithFrequentlyUsedEmojis(emojis: PickerEmojis): EmojiPickerL
     return addSpacesToEmojiCategories(mergedEmojis);
 }
 
-/**
- * Get the updated frequently used emojis list by usage
- */
-function getFrequentlyUsedEmojis(newEmoji: Emoji | Emoji[]): FrequentlyUsedEmoji[] {
-    let frequentEmojiList = [...frequentlyUsedEmojis];
-
-    const maxFrequentEmojiCount = CONST.EMOJI_FREQUENT_ROW_COUNT * CONST.EMOJI_NUM_PER_ROW - 1;
```

Comment:
We can remove `EMOJI_FREQUENT_ROW_COUNT` from the CONST file, as it is not used anywhere.